src/modules/wallet/selectors.ts

```diff
@@ -28,11 +29,11 @@ export const getNetwork = (state: any) =>
 export const getNetworks = (state: any) =>
   isConnected(state) ? getData(state)!.networks : undefined
 
+// This method is deprecated, it only returns the MANA balance on Ethereum, use getNetworks() to get the MANA balances on all the networks.
```

Comment:
If you add a `@deprecated` JSDOC tag, the method name will be crossed and the deprecation notice will be shown.
[Doc on how to do it](https://jsdoc.app/tags-deprecated.html)

Reply:
done 👍